src/Raven.Server/Rachis/RemoteConnection/RemoteConnection.cs

```diff
@@ -12,7 +12,7 @@
 using Sparrow.Logging;
 using Sparrow.Server.Utils;
 
-namespace Raven.Server.Rachis
+namespace Raven.Server.Rachis.RemoteConnection
```

Comment:
What is this change necessary?

Reply:
It just for better organization. 
But I will think about how to make it cleaner